keyboards/meletrix/zoom75/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2023 meletrix
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// Loaction { { col, row } )
+#define ENCODERS_CW_KEY  { { 3, 5 } }
+#define ENCODERS_CCW_KEY { { 4, 5 } }
```

Comment:
This bodge code for VIA to work with encoder's rotations shall be removed and use [encoder map](https://docs.qmk.fm/#/feature_encoders?id=encoder-map) for that instead, at that point this file would be empty and could be removed (afaik)
```suggestion
```